### ISSUE TITLE
Player:  Crash on startup.  pyglui -> IndexError: pop from empty list

### ISSUE THREAD

**Issue by performlabrit** [pupil-labs/pupil#1258]
OSX High Sierra and Player 1.7.42.

I'm exporting fixations from multiple files today, and I've run into this bug several times.  The quick fix is to delete 'user_settings_player' from pupil_player_settings in my home directory.  

```
2018-08-08 16:12:58,990 - player - [DEBUG] plugin: Loading plugin: Vis_Circle with settings {'radius': 20, 'color': [0.0, 0.7, 0.25, 0.2], 'thickness': 2, 'fill': True}
2018-08-08 16:12:58,991 - player - [ERROR] launchables.player: Process Player crashed with trace:
Traceback (most recent call last):
  File "launchables/player.py", line 420, in player
  File "pyglui/ui.pyx", line 256, in pyglui.ui.UI.configuration.__set__ (pyglui/ui.cpp:12585)
  File "pyglui/ui.pyx", line 248, in pyglui.ui.UI.set_submenu_config (pyglui/ui.cpp:12362)
  File "pyglui/menus.pxi", line 704, in pyglui.ui.Scrolling_Menu.configuration.__set__ (pyglui/ui.cpp:63029)
  File "pyglui/menus.pxi", line 87, in pyglui.ui.Base_Menu.set_submenu_config (pyglui/ui.cpp:50999)
  File "pyglui/menus.pxi", line 482, in pyglui.ui.Growing_Menu.configuration.__set__ (pyglui/ui.cpp:59207)
  File "pyglui/menus.pxi", line 87, in pyglui.ui.Base_Menu.set_submenu_config (pyglui/ui.cpp:50996)
IndexError: pop from empty list

2018-08-08 16:12:58,991 - player - [INFO] launchables.player: Process shutting down.
2018-08-08 16:13:06,997 - MainProcess - [INFO] os_utils: Re-enabled idle sleep.

```

**Comment by performlabrit** [comment]
As requested by @papr :  My log and (a zip of) my user_settings_player file.
 
[player.log](https://github.com/pupil-labs/pupil/files/2274125/player.log)
[user_settings_player.zip](https://github.com/pupil-labs/pupil/files/2274128/user_settings_player.zip)

**Comment by papr** [comment]
TL;DR: I was not able to reproduce the issue.

The general idea to reproduce the bug is to install @performlabrit 's custom plugin, replace the player settings with the uploaded settings file, and to start Player in hope that it crashes.

I tried:

1. Pupil Player, `v1.7-42`, macOS bundle, version from which the uploaded user settings were generated from
2. Pupil Player, current master, Linux from source. Note that I changed the settings version manually in order to prevent Player to discard the settings.

Both trials worked as expected and did not crash.

@performlabrit  To be honest, I think this bug is some weird pyglui interaction with your custom plugin. :-/ Also, after reviewing your ui code again, I did not find any issues with it.

**Comment by papr** [comment]
I think this issue can be closed since it is probably related to the custom plugin.
